kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KafClient.java

```diff
@@ -79,8 +82,9 @@ public void produceMessages(String topicName, String bootstrap, String message,
     @Override
     public List<ConsumerRecord> consumeMessages(String topicName, String bootstrap, int numOfMessages, Duration timeout) {
         LOGGER.atInfo().log("Consuming messages using kaf");
-        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL + "-kafka-go";
-        List<String> args = List.of("kaf", "-b", bootstrap, "consume", topicName, "--output", "json");
+        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL + "-kaf-" + MobyNamesGenerator.getRandomName().replace("_", "-");
+        List<String> args = List.of("kaf", "-b", bootstrap, "consume", topicName, "--output", "json", "--offset", String.valueOf(offset));
+        offset += numOfMessages;
```

Comment:
This makes no sense, if I consume three messages from topic A, then consume from topic B if it going to start at offset 3.

Reply:
I know, I'm making many tests, playing with offsets, commits, consumer groups and this shouldn't be committed to the branch.